### PR TITLE
generate stars

### PR DESCRIPTION
**Pull request by thehandsomepanther**

### CODE REVIEW

File: src/containers/app/index.js [hunk 1]
```diff
@@ -14,23 +11,13 @@ const renderIfSignedIn = (element, props) => (props.isLoggedIn && element);
...
-        </header>
-
-        <main>
-            <Route exact path="/" component={ Home } />
-            <Route exact path="/app" component={ Application } />
-            <Route exact path="/register" component={ Authentication } />
-            <Route exact path="/login" component={ Authentication } />
-            <Route exact path="/logout" component={ Logout }/>
-        </main>
-    </div>
```
Comment (brandonfujii) [inline src/containers/app/index.js]: where are we planning to render the navigation bar on pages that are not the home page?

File: src/containers/home/background.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
Comment (brandonfujii) [inline src/containers/home/background.js]: Do we need these imports?

File: src/containers/app/index.js (hunk 1 above)
Comment (thehandsomepanther) [reply src/containers/app/index.js]: do we need the navigation bar on other pages? the only place you would need to login button is on the home page, and the only time you would need to logout button is in the dashboard/application page, but tbh I'm not sure that people really need/would use the logout button

Comment (brandonfujii) [reply src/containers/app/index.js]: there are going to be other links like `/dashboard` (which should show the status of your application), `/application`, and `/talks`. how would we navigate to those pages?

**Comment by brandonfujii** [comment]
LGTM :+1:

**Pull request merged**